qemu/tests/cfg/numa_dist.cfg

```diff
@@ -0,0 +1,29 @@
+- numa_dist:
+    type = numa_dist
```

Comment:
Miss qemu restriction?

Reply:
> Miss qemu restriction?

Which version ? Could you add in in the case too ?
From my checking, `numa dist` was supported since qemu 2.9.0.

Reply:
Yeah, seems it's supported long time ago. I think it's ok without the restriction. Thanks.